index.js

```diff
@@ -25,7 +36,11 @@ function fastifySwagger (fastify, opts, next) {
   fastify.decorate('swaggerCSP', require('./static/csp.json'))
 }
 
-module.exports = fp(fastifySwagger, {
+const plugin = fp(fastifySwagger, {
   fastify: '>=3.x',
   name: 'fastify-swagger'
 })
+
+module.exports.default = plugin
```

Comment:
There is no need to specify `.default` or `.fastifySwagger`.
```suggestion
module.exports = plugin
module.exports.validatorCompiler = validatorCompiler
```

Reply:
I forgot to remove it, I will

Reply:
default is needed: https://github.com/fastify/fastify/blob/a04f4c952731c57e79c6ca3794a62959a43731ed/fastify.js#L707-L717

Reply:
@mcollina 
`.default` is not needed. It is added by `fastify-plugin` by default.
We do not need to re-export it.

// default export by fastify-plugin
https://github.com/fastify/fastify-plugin/blob/master/plugin.js#L10-L13
// named export by fastify-plugin
https://github.com/fastify/fastify-plugin/blob/master/plugin.js#L59-L62